stingray/events.py

```diff
@@ -365,71 +374,60 @@ def join(self, other):
         return ev_new
 
     @staticmethod
-    def read(filename, format_='pickle'):
+    def read(filename, format_="pickle", **kwargs):
```

Comment:
Remind me again why there's a `_` on the `format` keyword?

Reply:
I think that `format` is a protected word in Python. Usman chose this syntax, we might change it into fmt or something similar, but I didn't want to break the API just now.

Reply:
ah, yeah, that makes sense. Let's leave it and not break the syntax